Extraction/SalesforceCustomExtractionUtility/chunks_extraction.py

```diff
@@ -31,19 +37,79 @@
     "timeout":150,
     "max_retries":0
 }
+azure_open_ai_conf = {
+    "API_BASE": "https://koreopenai.openai.azure.com",
```

Comment:
@hariprasadkothuru Can you please externalise "API_BASE" and "model"as well and read from the config.